packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -256,7 +256,7 @@ mixin WidgetsBinding on BindingBase, SchedulerBinding, GestureBinding, RendererB
     window.onAccessibilityFeaturesChanged = handleAccessibilityFeaturesChanged;
     SystemChannels.navigation.setMethodCallHandler(_handleNavigationInvocation);
     SystemChannels.system.setMessageHandler(_handleSystemMessage);
-    FlutterErrorDetails.offendingContextDescribers.add(describeOffendingContext);
+    FlutterErrorDetails.informationTransformers.add(transformOffendingContext);
```

Comment:
nit: perhaps call the transformer: computeOffendingContext as you are deriving the offending context from the debugCreator.